src/pages/BusinessSpecifics/BusinessSpecifics.tsx

```diff
@@ -2,14 +2,35 @@ import { FormControl, TextField, Button, MenuItem, Select } from "@mui/material"
 import { useState } from "react"
 import { currencies } from "./currencies"
 
-export const BusinessSpecifics = () => {
 
-    const [BusinessName, setBusinessName] = useState("")
-    const [LocationName, setLocationName] = useState("")
+export const BusinessSpecifics = () => {
+    const url = 'https://invent0ry-back3nd.herokuapp.com/businesses/'
+    const [businessName, setBusinessName] = useState("")

```

Comment:
usually we just need one "useState" hook with an object.
`const [values, setValues] = useState<valuesTypes>({businessName:"", locationName:"", currency:""})`

Reply:
We also need to specify the "valuesTypes" using typescript syntax

Reply:
because the hook is representing the whole business form, we combine it into one?